packages/api/src/routes/medical/__tests__/fixtures.ts

```diff
@@ -0,0 +1,67 @@
+export function makePatientData(
+  overrides: Partial<{
+    firstName: string;
+    lastName: string;
+    dob: string;
+    contact: Array<{ phone: string }>;
+    address: Array<{ state: string }>;
+  }> = {}
+) {
+  return {
+    firstName: "John",
+    lastName: "Doe",
+    dob: "1990-01-01",
+    contact: [{ phone: "555-1234" }],
+    address: [{ state: "CA" }],
+    ...overrides,
+  };
+}
+
+export function makePatient(
+  overrides: { patientData?: Partial<ReturnType<typeof makePatientData>> } & Record<
+    string,
+    unknown
+  > = {}
+) {
+  const patientData = makePatientData(overrides.patientData);
+  return {
+    id: "patient-123",
+    cxId: "cx-123",
+    data: patientData,
+    dataValues: {
+      id: "patient-123",
+      cxId: "cx-123",
+      data: patientData,
+    },
+    ...overrides,
+  };
+}
+
+export function makeEncounter(
+  overrides: { patient?: Partial<ReturnType<typeof makePatient>> } & Record<string, unknown> = {}
+) {
+  const patient = makePatient(overrides.patient);
+  const baseEncounter = {
+    id: "enc-1",
+    cxId: "cx-123",
+    patientId: "patient-123",
```

Comment:
Let's use valid data as much as possible, in this case UUIDs.

Also, while making tests deterministic is important, things that should be supported with dynamic values is good to build them as such - see `faker.string.uuid()`

Reply:
Flagged initially [here](https://github.com/metriport/metriport/pull/4098#discussion_r2169514896)